docs/multicluster/quick-start.md

```diff
@@ -0,0 +1,172 @@
+# Antrea Multi-cluster Quick Start
+
+In this quick start guide, we will set up an Antrea Multi-cluster ClusterSet
+with two clusters. One cluster will serve as the leader of the ClusterSet, and
+meanwhile also join as a member cluster; another cluster will be a member only.
+
+The diagram belows shows the two clusters and the ClusterSet to be created (for
+simplicity, the diagram just shows two Nodes for each cluster).
+
+<img src="assets/sample-clusterset.svg" width="800" alt="Antrea Multi-cluster Example ClusterSet">
+
+## Preparation
+
+We assume an Antrea version >= v1.7.0 is used in this guide, and the Antrea
+version is set to an environment variable `TAG`. For example, the following
+command sets the Antrea version to v1.7.0.
+
+```bash
+export TAG=v1.7.0
+```
+
+To use the latest version of Antrea Multi-cluster from the Antrea main branch,
+you can change the YAML manifest path to: https://github.com/antrea-io/antrea/tree/main/multicluster/build/yamls/.
+
+Antrea must be deployed in both cluster A and cluster B. To configure Antrea
+Multi-cluster Gateways, `antrea-agent` must be deployed with the `Multicluster`
+feature enabled in both cluster A and B. Multi-cluster Gateways are required for
+routing multi-cluster Service traffic through tunnels across the member
+clusters. Set the following options in `antrea-agent.conf` of the Antrea
+deployment manifest to enable the `Multicluster` feature:
+
+```yaml
+antrea-agent.conf: |
+...
+  featureGates:
+...
+    Multicluster: true
+...
+  multicluster:
+    enable: true
+    namespace: ""
+```
+
+## Set up Leader and Member in Cluster A
+
+### Step 1 - deploy Antrea Multi-cluster Controllers for leader and member
+
+Run the following commands to deploy Multi-cluster Controller for the member
+Namepsace `kube-system`, and  Multi-cluster Controller for the leader into
+Namespace `antrea-multicluster` (Namespace `antrea-multicluster` will be created
+by the commands):
+
+```bash
+$kubectl apply -f https://github.com/antrea-io/antrea/releases/download/$TAG/antrea-multicluster-member.yml
+$kubectl apply -f https://github.com/antrea-io/antrea/releases/download/$TAG/antrea-multicluster-leader-global.yml
```

Comment:
Do we need to apply antrea-multicluster-leader-global.yml for a dual-role cluster? In the user guide, earlier you wrote it is necessary, as antrea-multicluster-member.yml already import all CRDs.

If we do need to apply all YAMLs and the order does not matter, I would apply leader YAMLs first, which is easier to understand. @luolanzone

Reply:
yes, we removed those unnecessary CRDs in leader yamls, so we need to apply antrea-multicluster-leader-global.yml for a dual-role cluster, I will move it.

Reply:
Done, the diagram is also updated.